### DISCUSSION TITLE
Card animations sometimes don't work

### DISCUSSION THREAD

**Discussion by GermanBartoli** [anuraghazra/github-readme-stats#3546] [Q&A] [answered]
### Describe the bug

Sometimes the image doesn't load. I had to disable the animation for it to work. Idk how to explain it, I upload a photo. For the first time it doesn't work, after a refresh it work.

<a
    href="https://wakatime.com/@bartoligerman497"
    target="_blank"
    rel="noreferrer">
    <img
        src="https://github-readme-stats.vercel.app/api/wakatime?username=bartoligerman497&langs_count=5&hide=other,binary&hide_border=true&title_color=0366d6&text_color=777777&icon_color=959da5&bg_color=ffffff00"
        width="100%"
        height=""
        alt="WakaTimeStats"
    />
 </a>

<a
   href="https://github.com/bartoligerman497"
   target="_blank"
   rel="noreferrer">
    <img
        src="https://github-readme-stats.vercel.app/api?username=bartoligerman497&count_private=true&show_icons=true&hide_border=true&title_color=0366d6&text_color=777777&icon_color=959da5&bg_color=ffffff00"
        width="100%"
        height=""
        alt="ProfilePoints"
    />
</a>

### Screenshots / Live demo link

![image](https://github.com/anuraghazra/github-readme-stats/assets/53313625/dbf5428a-c5f8-450d-bcbe-892697f5d2f3)

**Comment by qwerty541** [comment, marked as answer]
Hey, @bartoligerman497! Thanks for your report! Recently we discussed this issue here https://github.com/anuraghazra/github-readme-stats/issues/3487 and came to conslusion that it cannot be handled from our side. Most likely these animations freezings was caused by some browser optimizations. If you experience this too often you can use `disable_animations` query parameter to avoid it.
